### ISSUE TITLE
move.random executes collisions incorrectly

### ISSUE THREAD

**Issue by voidshaper777** [XXIIVV/Blind#19]
move.random currently can trigger a collision event ('Blindfolk x counters!' message is displayed) but no scripted commands for the corresponding case will be executed.

**Comment by neauoire** [comment]
Can you confirm that this is still the case?

**Comment by voidshaper777** [comment]
Yes, confirmed. Attack responses trigger correctly, collision responses not.
